draft-ietf-ppm-dap.md

```diff
@@ -1500,16 +1498,14 @@ Mandatory validation of extensions will occur during aggregation.)
 
 ### Report Extensions {#report-extensions}
 
-Each `ReportMetadata` contains a list of extensions public to both aggregators,
-and each `PlaintextInputShare` contains a list of extensions private to the
-relevant Aggregator. Clients use these extensions to convey additional
-information to the Aggregators. Some extensions might be intended for both
-Aggregators; others may only be intended for a specific Aggregator. (For
-example, a DAP deployment might use some out-of-band mechanism for an Aggregator
-to verify that reports come from authenticated Clients. It will likely be useful
-to bind the extension to the input share via HPKE encryption.)
+Clients use report extensions to convey additional information to the
+Aggregators. Each `ReportMetadata` contains a list of extensions public to both
+aggregators, and each `PlaintextInputShare` contains a list of extensions
+private to the relevant Aggregator. For example, Clients may need to
+authenticate to the Helper by presenting a secret that must not be revealed to
+the Leader.
```

Comment:
Brevity: remove parenthetical that anticipates some future document defining a future extension